### ISSUE TITLE
River Mile Markers

### ISSUE THREAD

**Issue by kechinn** [wsdot-eso/Environmental-Information#61]
**Description:** Check if this needs update/what the plan is for future updates.
**Category:** Extnrl SDE Data - Update
**Workspace:** 
**% Complete:** 20%

**Comment by kechinn** [comment]
Fuelling, Garrett (12/28/2020 4:50 PM): This seems to have had an update in Nov of 2014. https://fortress.wa.gov/ecy/gispublic/DataDownload/ECY_WAT_USGS_WDFW_RiverMiles.htm Slightly different naming but I believe it is the same dataset

**Comment by gfuellingwa** [comment]
Change name in WB from River Mile Benchmarks to River Mile Markers

**Comment by kechinn** [comment]
Just FYI, got an email regarding this:
From: Briscoe, VaLoy <BriscoV@wsdot.wa.gov> 
Sent: Monday, January 25, 2021 11:45 AM
To: Mitchell, Jordyn <MitcheJo@wsdot.wa.gov>; Lanzer, Elizabeth <LanzerE@wsdot.wa.gov>
Subject: Possible Workbench river mile markers data update.

Hi Jordyn and Elizabeth,

Ecology updated their river mile markers data in 2014.
 
Summary 
Show Washington State river mile locations for hydrology projects.
Description 
Ecology created the GIS statewide river mile point layer in March 2007 by digitizing the river mile points depicted on the USGS 7½ minute (24k) topographic quadrangle maps. Some of the rivers have gaps in the river mile progression because several of the quadrangle maps do not have any river mile points, while a few were missing a point or two. In November 2014 Ecology added river mile points for the missing areas using Washington Department of Fish and Wildlife's (WDFW) 1975 Stream Catalog, which only covers WRIA's 1 through 24. The Stream Catalog shows river miles for nearly every stream; however, only those water courses that have river miles from the USGS quadrangle maps were added. The field SOURCE denotes the source of the data point, USGS or WDFW. Discrepancies between the USGS and WDFW are documented in the Supplemental Information section.
Credits 
Washington State Department of Ecology

I could only find the older data on the Workbench so I downloaded the .gdb from the Ecology website. 

I don’t use the Workbench very often, so my apologies if I missed it.

Hope your week goes well,
VaLoy


VaLoy Briscoe
GIS Specialist
WSDOT – South Central Region
2809 Rudkin Road
Union Gap, WA 98903-1648
Telework:   (509) 833-9462
Office:         (509) 577-1950
